app/services/ecr_service.rb

```diff
@@ -0,0 +1,45 @@
+class EcrService
+  def validate_image!(heritage)
+    @heritage = heritage
+    repository_name = @heritage.image_name[%r{/([^/]*?)$}, 1]
```

Comment:
what does this do?

Reply:
added the comments
let me know if this is still unclear.